Vic2ToHoI4/Source/HOI4World/HoI4World.cpp

```diff
@@ -1341,7 +1341,7 @@ void HoI4::World::outputBookmarks() const
 	}
 	
 	bookmarkFile << "		\"---\"= {\n";
-	bookmarkFile << "			history = \"OTHER_GATHERING_STORM_DESC\"\n";
+	bookmarkFile << "			history = \"" + greatPower->getGovernmentIdeology() + "_GP_CONV_DESC\"\n";
```

Comment:
Did you maybe intend to change line 1337 above instead?

Reply:
No, that one I replaced it. I modded ideology-related GP and SP descriptions, which are better than having a text talk about the situation of the world even after you selected Germany.